activerecord/lib/active_record/relation/batches.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_support/deprecation'
```

Comment:
Is this require necessary? I think `ActiveSupport::Deprecation` is available after `active_support` has been required:

```
irb(main):001:0> require 'active_support'
=> true
irb(main):002:0> ActiveSupport::Deprecation
=> ActiveSupport::Deprecation
```